nibabel/gifti/tests/test_gifti.py

```diff
@@ -425,11 +425,12 @@ def test_gifti_coord(capsys):
     captured = capsys.readouterr()
     assert (
         captured.out
-        == """Dataspace:  NIFTI_XFORM_UNKNOWN
+        == """\
+Dataspace:  NIFTI_XFORM_UNKNOWN
 XFormSpace:  NIFTI_XFORM_UNKNOWN
-Affine Transformation Matrix: 
```

Comment:
This makes me think that the intent was to have:
```
Dataspace:  NIFTI_XFORM_UNKNOWN
XFormSpace:  NIFTI_XFORM_UNKNOWN
Affine Transformation Matrix:  None
```
instead of:
```
Dataspace:  NIFTI_XFORM_UNKNOWN
XFormSpace:  NIFTI_XFORM_UNKNOWN
Affine Transformation Matrix: 
 None
```
@effigies What do you think?

Reply:
No, then intent was for the affine to appear on the next line, when present:

```
Coordinate System:
Dataspace:  NIFTI_XFORM_SCANNER_ANAT
XFormSpace:  NIFTI_XFORM_UNKNOWN
Affine Transformation Matrix:
 [[  1.         0.         0.         5.255989]
 [ -0.         1.        -0.       -13.557167]
 [ -0.        -0.         1.        26.079163]
 [  0.         0.         0.         1.      ]]
```

Possibly in the first submission it did start on the same line, but it definitely looks more reasonable on the next line.